core/merginapi.cpp

```diff
@@ -1347,14 +1347,8 @@ void MerginApi::checkMerginVersion( QString apiVersion, bool serverSupportsSubsc
   {
     int major = -1;
     int minor = -1;
-    QRegularExpression re;
-    re.setPattern( QStringLiteral( "(?<major>\\d+)[.](?<minor>\\d+)" ) );
-    QRegularExpressionMatch match = re.match( apiVersion );
-    if ( match.hasMatch() )
-    {
-      major = match.captured( "major" ).toInt();
-      minor = match.captured( "minor" ).toInt();
-    }
+
+    parseVersion( apiVersion, major, minor );
```

Comment:
Would it be possible to add server version + flavour (EE/CE/SaaS) to the "sticky" part of the diagnosis log https://github.com/MerginMaps/input/blob/780622dacf156cad443caa5fa6fd0089c21e2ae9/app/inputhelp.cpp#L184 ?

Reply:
Added server version + flavour (EE/CE/SaaS) to the diagnosis log